.editorconfig

```diff
@@ -4,71 +4,24 @@
 
 root = true
 
-[*.php]
+[*.{js,php,py,yml}]
```

Comment:
I believe we'd split this up because there was poor support at the time for this format. Does this work well now?